remote-shell-doc/remote-shell.scrbl

```diff
@@ -209,3 +210,21 @@ Returns the UUID of @racket[snapshot-name] for the virtual machine
 @racket[name].
 
 @history[#:added "1.1"]}
+
+@defproc[(import-vbox-vm [path path-string?]
+                         [#:name name (or/c false/c non-empty-string?) #f]
+                         [#:cpus cpus (or/c false/c exact-positive-integer?) #f]
+                         [#:memory memory (or/c false/c exact-positive-integer?) #f]) void?]{
+
+Imports a VirtualBox VM from the OVF file at @racket[path].
+
+When provided, @racket[name] specifies what the VM's alias (as seen in
+the GUI and in the output of commands like @exec{VBoxManage list vms})
+ought to be.
+
+@racket[cpus] can be used to override the number of processors the VM
```

Comment:
Minor style suggestion: `The @racket[cpus] argument` instead of just `@racket[cpus]` to avoid starting a sentence with a lowercase identifier.